rails_programming/introduction_to_rails/project_feet_wet.md

```diff
@@ -8,17 +8,17 @@ Next comes the [Jumpstart Lab Blogger Tutorial](http://tutorials.jumpstartlab.co
 
 We will be using the Ruby on Rails Tutorial by Michael Hartl throughout this course.  It is a complete, step-by-step walkthrough of building and testing a Twitter-like microblogging application.  It covers a lot of material but so does this course, so we'll be spacing out the chapters along the way to go with the appropriate lesson material.
 
-Though this tutorial is a major common theme across all the coming lessons and serves to tie them together into one major project, we'll also be building plenty of independent projects along the way.
+Through this tutorial is a major common theme across all the coming lessons which serves to tie them together into one major project. We'll also be building plenty of independent projects along the way.
 
-In this chapter, you'll build a simple scaffolded app on Rails.  It won't be as complicated as the Blogger Tutorial was and is really just to make sure your system is all set up and your brain is switched into Rails mode.
+In this chapter, you'll build a simple scaffolded app on Rails.  It won't be as complicated as the Blogger Tutorial was, and is really just to make sure your system is all set up and your brain is switched into Rails mode.
```

Comment:
This comma is incorrect and shouldn't be added. 

(If you want to know why, it's because adding a comma there would separate the subject, "It", from its verb, "is". This is an example of a sentence with a compound predicate; it is *not* a compound sentence.)